ihmc-model-file-loader/src/main/java/us/ihmc/modelFileLoaders/SdfLoader/RobotDescriptionFromSDFLoader.java

```diff
@@ -636,7 +637,7 @@ private void addLidarMounts(SDFSensor sensor, JointDescription scsJoint, SDFLink
             }
             else
             {
-               System.err.println("Unknown noise model: " + sdfNoise.getType());
+               LogTools.info("Unknown noise model: " + sdfNoise.getType());
```

Comment:
No the case here, but it reminded me, careful doing this more broadly. Anything that executes as part of a Runtime shutdownHook can't use LogTools anymore, because log4j2 is one of the things that gets shutdown in that phase.

Reply:
No clue, if that's the case I can cahnge it back.